libs/platform/experience/src/routes.ts

```diff
@@ -30,6 +30,13 @@ export const defaultExperienceRoutes: RouteConfig[] = [
     ></oryx-composition>`,
     name: 'Order',
   },
+  {
+    path: '/article/:id',
```

Comment:
We do want to end up with various entities connected to various cms providers, i.e.:
- `/faq/:id` - could be resolved by cms 1
- `/jobs/:id` - could be resolved by cms 2
- `/:id` - fallback content, could be resolved by cms 3

Totally fine to work on this going forward, but want to make sure we're aligned on the vision so you can make the right choices in the design.